crates/compiler/load_internal/src/file.rs

```diff
@@ -5053,13 +4799,252 @@ fn fabricate_platform_module<'a>(
         imports: unspace(arena, header.imports.item.items),
     };
 
-    send_header_two(
-        info,
-        parse_state,
-        module_ids,
-        ident_ids_by_module,
-        module_timing,
-    )
+    {
+        let PlatformHeaderInfo {
+            filename,
+            opt_shorthand,
+            is_root_module,
+            opt_app_module_id,
+            packages,
+            provides,
+            requires,
+            requires_types,
+            imports,
+        } = info;
```

Comment:
Could this defer to `send_header`, since `send_header_two` is now rolled up into there?

Reply:
I see, this was lifted in another commit